addons/godot-xr-tools/assets/PlayerBody.gd

```diff
@@ -94,11 +94,43 @@ class SortProviderByOrder:
 	static func sort_by_order(a, b) -> bool:
 		return true if a.order < b.order else false
 
+# Get our origin node, make sure we have consistent code here
+func _get_origin_node() -> ARVROrigin:
+	var node : ARVROrigin = get_node_or_null(origin) if origin else get_parent()
+	return node
```

Comment:
So I feel this should always return the parent, we should remove the origin export